### PR TITLE
Init project

### PR DESCRIPTION
**Pull request by skhomuti**
- add nodes package
- add wallets-testing project
- add workflows

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+          node-version: '16'
+          cache: 'yarn'
+      - name: Install dependencies
+        run: yarn install --immutable
+      - name: Run lint
+        run: yarn lint
+      - name: Build Components
+        run: yarn build
+
+      - name: Install dependencies
```
Comment (dgusakov) [inline .github/workflows/deploy.yml:28]:
```suggestion
      - name: Install dependencies in wallets-testing
```

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      - name: Run walllets tests
+        run: yarn test
+        working-directory: wallets-testing
+        env:
+          RPC_URL: ${{ secrets.RPC_URL }}
+#      - name: Publish to NPM
+#        env:
+#          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+#          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+#        run: yarn multi-semantic-release --deps.bump=override --deps.release=patch --sequential-init
```
Comment (dgusakov) [inline .github/workflows/deploy.yml:40]: why commented?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,21 @@
-# wallets-testing-modules
-Modules for testing wallets integrations
+# Lido NestJS Modules for wallets testing
+
+NestJS modules for automated testing wallets integrations.
```
Comment (dgusakov) [inline README.md:3]:
```suggestion
NestJS modules for wallets integrations automated testing
```

File: packages/nodes/README.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+})
+export class MyModule {}
+
+// Usage
+export class MyService {
+  constructor(private ethereumNodeService: EthereumNodeService) {}
+
+  async myMethod() {
+     await this.ethereumNodeService.startNode();
+     const state = await this.ethereumNodeService.state;
```
Comment (dgusakov) [inline packages/nodes/README.md:35]: No state assertion

File: packages/nodes/src/ethereum.node.service.ts [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+  public static readonly stakeAmount = '100';
+
+  state: { node: Server; account: string; secretKey: string } | undefined;
+
+  constructor(@Inject(OPTIONS) private options: { rpcUrl: string }) {}
+
+  async startNode() {
+    if (this.state !== undefined) return;
+    const node = ganache.server({
+      chainId: 0x1,
```
Comment (dgusakov) [inline packages/nodes/src/ethereum.node.service.ts:28]: In future, it will be great to make it configurable

Comment (skhomuti) [reply packages/nodes/src/ethereum.node.service.ts]: done

File: packages/nodes/README.md (hunk 4 above)
Comment (skhomuti) [reply packages/nodes/README.md:35]: added

File: .github/workflows/deploy.yml (hunk 2 above)
Comment (skhomuti) [reply .github/workflows/deploy.yml:40]: there's no npm tokens at the moment, will uncomment it a bit late

**Review by dgusakov — APPROVED** [review]

**Pull request merged**